keps/sig-node/4680-add-resource-health-to-pod-status/README.md

```diff
@@ -368,6 +376,10 @@ message DeviceHealth {
   // Timestamp of when this health status was last determined by the plugin, as a Unix timestamp (seconds).
   // Required.
   int64 last_updated_timestamp = 4;
+  // Health check timeout duration in seconds for this device.
+  // If not specified or zero, Kubelet will use a default timeout.
+  // Optional.
+  int64 health_check_timeout_seconds = 5;
```

Comment:
What would the negative value or `0` mean? Let's specify in the field description.